test/e2e-test-application/src/app/project/settings/settings.component.html

```diff
@@ -91,5 +91,21 @@ <h3 class="fd-layout-panel__title">
         to go to <i>project two</i>, where you can try out hasBack functionality
       </div>
     </div>
+    <div class="fd-layout-panel">
+      <div class="fd-layout-panel__header">
+        <div class="fd-layout-panel__head">
+          <h3 class="fd-layout-panel__title">
+            LuigiClient Modal
+          </h3>
+        </div>
+      </div>
+      <div class="fd-layout-panel__body">
+        <div *ngIf="isModal">
+          <button class="fd-button" (click)="uxManager().closeModal()">
+            Close modal
+          </button>
+        </div>
```

Comment:
```suggestion
          <button class="fd-button" (click)="uxManager().closeModal()">
            Close modal
          </button>
```

Just part 2 of the first suggestion.